<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QLineEdit" name="GuessInput">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="GuessLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Guess:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreLabel">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Score: </string>
   </property>
  </widget>
  <widget class="QLabel" name="CorrectLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Correct/Incorrect</string>
   </property>
  </widget>
  <widget class="QLabel" name="LetterBankLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Win/Lose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Timer">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>GUESS</string>
   </property>
  </widget>
  <widget class="QLabel" name="HeadLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="bodyArmsLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/|\</string>
   </property>
  </widget>
  <widget class="QLabel" name="legsLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> /\</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>71</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>  _______
 |/      |
 |      
 |      
 |      
 |
_|___</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
